doc/script_commands.txt

```diff
@@ -11102,14 +11101,8 @@ Returns 1 on success.
 Set option for the channel. Use 1 in <value> to set it, or 0 to unset.
 The <option> values are the same as the 'channel_create' options.
 
-For CHAN_OPT_MSG_DELAY, the delay in millisecond must be sent or use 0
-to remove the delay at <value>.
-
```

Comment:
This is not reflected in source.
Maybe you just wanted to rename the constant here?